### ISSUE TITLE
[QA] Modern linux packages based on AppImage (5.x+)

### ISSUE THREAD

**Issue by TheOneRing** [owncloud/client#10702]
Blocks:
- https://github.com/owncloud/client/issues/10701 

Development:
- https://github.com/owncloud/client/issues/10962

Full working repos should be available latest with the 5.0.0-rc.1 (2023-08-25) or earlier in a new 5.0.0-beta.x

### Vanilla Installation + Update

1. Install 4.1 client on the relevant supported Linux distros:
  https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/download/
2. Follow 4.1 docs:
  https://doc.owncloud.com/desktop/4.1/installing.html#installation-on-linux
3. Then install 5.0 client:
  https://download.owncloud.com/desktop/ownCloud/testing/5.0.0.xxxxxx-rc.1/linux/download/
4. Follow 5.0 docs: https://doc.owncloud.com/desktop/next/installing.html#installation-on-linux

### Vanilla Testing Steps?

- Install errors
- Does everything work?
- Shell extensions: context menu + overlay icons
- install test script

## Branded Testing

Basically, test a branded (e.g., QA theme) client upgrade to make sure customer deployment still works. See also https://doc.owncloud.com/branded_clients/next/desktop_app/linux_deployment.html.

1. Deploy branded 4.1 repo similar to https://cloud.damken.com/install/linux/download/
  Docs: https://doc.owncloud.com/branded_clients/next/desktop_app/linux_deployment.html
2. Install branded 4.1 client on the relevant supported Linux distros
  - most relevant deb-based distro (Ubuntu 22.04)
  - most relevant rpm-based distro (latest stable Fedora)
4. Deploy branded 5.0 repo on the same location
5. Execute `download/example.sh`
6. Then install 5.0 client

### Branded Testing Steps?

- Install errors
- Does everything work?
- Shell extensions: context menu + overlay icons
- install test script

**Comment by fmoc** [comment]
Proof of concept done, builds on GitHub actions are on the way.

**Comment by fmoc** [comment]
We need another minor change in the blueprint, then we should have artifacts. I'll make the commit ASAP.

**Comment by michaelstingl** [comment]
Superseded by:
- https://github.com/owncloud/client/issues/10962

**Comment by fmoc** [comment]
@HanaGemela please run some short tests with the new packages. We need to make sure the packages work on our supported distros (Ubuntu, Debian, Fedora, CentOS 7). Please report all bugs as new issues.

CC @michaelstingl

**Comment by HanaGemela** [comment]
Vanilla install failed on Fedora 37: https://github.com/owncloud/client/issues/11130

**Comment by Salipa-Gurung** [comment]
Vanilla installation passed in CentOs 7
For fresh installation I had to run command `yum install libglvnd-opengl`

**Comment by saw-jan** [comment]
Branded client (Testpilot): `testpilotcloud-client-5.0.0.11948_daily20230829`

`Debian 11` :x: : Same error as vanilla upgrade (https://github.com/owncloud/client/issues/11133)
`Fedora 28` :x: : Same error as vanilla upgrade (https://github.com/owncloud/client/issues/11133)

**Comment by fmoc** [comment]
The next set of dailies, scheduled for publishing around 2pm CEST (in ~10 minutes) should fix both issues. I'll have to test it myself, though.

**Comment by saw-jan** [comment]
Vanilla upgrade: (`ownCloud 5.0.0.11982-daily20230831 dfca9f`)

- `Debian 11`: :heavy_check_mark:
- `Fedora 37`:

**Comment by saw-jan** [comment]
Branded upgrade: ()

- `Debian 11`: 
- `Fedora 38`:

**Comment by HanaGemela** [comment]
> Branded upgrade: ()
> 
> * `Debian 11`:
> * `Fedora 38`:

Please follow the steps at the top:
most relevant deb-based distro (Ubuntu 22.04)
most relevant rpm-based distro (latest stable Fedora)

**Comment by HanaGemela** [comment]
QA coordinated in https://github.com/owncloud/client/issues/11171

**Comment by fmoc** [comment]
Done development wise. QA needs to approve.

**Comment by HanaGemela** [comment]
QA approved here https://github.com/owncloud/client/issues/10702#issuecomment-1706744699
